peps/pep-0747.rst

```diff
@@ -711,12 +735,38 @@ assigned to variables and manipulated like any other data in a program:
 ``TypeExpr[]`` is how you spell the type of a variable containing a
 type annotation object describing a type.
 
-``TypeExpr[]`` is similar to ``type[]``, but ``type[]`` can only used to
+``TypeExpr[]`` is similar to ``type[]``, but ``type[]`` can only
 spell simple **class objects** like ``int``, ``str``, ``list``, or ``MyClass``.
 ``TypeExpr[]`` by contrast can additionally spell more complex types, 
 including those with brackets (like ``list[int]``) or pipes (like ``int | None``),
 and including special types like ``Any``, ``LiteralString``, or ``Never``.
 
+A ``TypeExpr`` variable looks similar to a ``TypeAlias`` definition, but
+can only be used where a dynamic value is expected.
```

Comment:
I don't understand what "only be used where a dynamic value is expected"? What is a "dynamic value" in this context?